python/servo/post_build_commands.py

```diff
@@ -263,11 +263,16 @@ def package(self, release=False, dev=False, debug=False, debugger=None):
     @CommandArgument('--dev', '-d', action='store_true',
                      help='Package the dev build')
     def install(self, release=False, dev=False):
-        binary_path = self.get_binary_path(release, dev, android=True)
-        if not path.exists(binary_path):
-            # TODO: Run the build command automatically?
-            print("Servo build not found. Please run `./mach build` to compile Servo.")
-            return 1
+        try:
+            binary_path = self.get_binary_path(release, dev, android=True)
+        except BuildNotFound:
+            print("Servo build not found. Building servo...")
+            result = Registrar.dispatch(
+                "build", context=self.context, release=release, dev=dev
+            )
+            if result:
+                return result
+            return self.install(release=release, dev=dev)
```

Comment:
If someone is doing development work and changes the build process to place its results somewhere other than the location that `get_binary_path` expects, this would loop indefinitely. Calling `get_binary_path` explicitly here and exiting with an explanation ("rebuilding Servo failed to solve the missing build problem") would save the hypothetical future novice some debugging headaches, and wouldn't impact performance.